ui.content.sample/src/main/content/jcr_root/content/wknd/ca/en/errors/.content.xml

```diff
@@ -31,4 +31,5 @@
     </jcr:content>
     <_x0034_04/>
     <_x0035_00/>
+    <sign-in/>
```

Comment:
Would it be better to have this be named 40x? (401, 403? ..) And maybe give the page title as Sign in?

Looks strange having sign-in under "errors"? Or should it even be under errors?